BoxContentSDK/BoxContentSDK/Operations/BOXStreamOperation.m

```diff
@@ -148,7 +148,9 @@ - (long long)contentLength
 
 - (void)abortWithError:(NSError *)error
 {
+    self.error = error;
     [self.sessionTask cancel];
+    //FIXME: Remove?
```

Comment:
Minor: let's add the date and author next to the FIXME. This adds context for future developers.